Update "multiple schedulers" example (#30107)
kubernetes/website: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md

{{< codenew file="admin/sched/my-scheduler.yaml" >}}

> In the above manifest, you use a [Scheduler Configuration](/docs/reference/scheduling/config/)

Reviewer: Is it better to refer to Scheduler Configuration as KubeSchedulerConfiguration?
If Scheduler Configuration is not an API object then it should be lower case

Author: @reylejano IMO, it is better to use `KubeSchedulerConfiguration` as this is a API resource type although it is not a server resource type.

@alculquicondor @damemi please correct me if I am wrong.

Author: @reylejano IMO, it is better to use `KubeSchedulerConfiguration` as this avoids any confusion regarding the resource type (this is an API resource type although it is not a server resource type). @alculquicondor @damemi WDYT?

Reviewer 2: I usually try to use `KubeSchedulerConfiguration` to show that I am specifically talking about this file/API object (and not, for example, other types of config like flags/policy/etc...)

Reviewer 3: `KubeSchedulerConfiguration` sgtm

Author: @damemi @alculquicondor ack. Will update.

Author: @reylejano updated in https://github.com/kubernetes/website/pull/30107/commits/7449bb36a0eaa7f2983fa8fae005e00f06496fa8.

> to customize the behavior of your scheduler implementation. This configuration has been passed to

Reviewer 3: mention the ConfigMap and the volume mount

the `kube-scheduler` during initialization with the `--config` option. The `my-scheduler-config` ConfigMap stores the configuration file. The Pod of the`my-scheduler` Deployment mounts the `my-scheduler-config` ConfigMap as a volume.

> In the aforementioned Scheduler Configuration, your scheduler implementation has been represented via
> a [KubeSchedulerProfile](/docs/reference/config-api/kube-scheduler-config.v1beta2/#kubescheduler-config-k8s-io-v1beta2-KubeSchedulerProfile).
> An important thing to note here is that the name of the scheduler specified via `schedulerName` field of the defined `KubeSchedulerProfile`,
> should be unique among all schedulers running in the cluster. This is the name that is matched against the value of the optional `spec.schedulerName` on pods, to determine whether
> this scheduler is responsible for scheduling a particular pod.
> 
> Also, note that we created a dedicated service account `my-scheduler` and bound the cluster role

Reviewer: ```suggestion
In the aforementioned Scheduler Configuration, your scheduler implementation is represented via
a [KubeSchedulerProfile](/docs/reference/config-api/kube-scheduler-config.v1beta2/#kubescheduler-config-k8s-io-v1beta2-KubeSchedulerProfile).
{{< note >}}
To determine if a scheduler is responsible for scheduling a specific Pod, the `spec.schedulerName` field in a 
PodTemplate or Pod manifest must match the `schedulerName` field of the `KubeSchedulerProfile`.
All schedulers running in the cluster must have unique names.
{{< /note >}}

Also, note that you create a dedicated service account `my-scheduler` and bind the ClusterRole
```

Maybe use the note shortcode to highlight the content

{{< codenew file="admin/sched/my-scheduler.yaml" >}}

> In the above manifest, we have used a [Scheduler Configuration](/docs/reference/scheduling/config/)

Reviewer: ```suggestion
In the above manifest, you use a [Scheduler Configuration](/docs/reference/scheduling/config/)
```
Under [Content best practices](https://kubernetes.io/docs/contribute/style/style-guide/#content-best-practices) in the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) suggests to use present tense and to address the reader as you not we

to customize the behavior of your scheduler implementation. This configuration has been passed to

> the `kube-scheduler` during its initialization via `--config` command line argument. The configuration
> file has been embedded within the ConfigMap `my-scheduler-config` which has been injected to the scheduler
> Deployment via a volume.

Reviewer: ```suggestion
the `kube-scheduler` during initialization with the `--config` option. The `my-scheduler-config` ConfigMap stores the configuration file. The Pod of the`my-scheduler` Deployment mounts the `my-scheduler-config` ConfigMap as a volume.
```
nit: for tense

To run multiple-scheduler with leader election enabled, you must do the following:

> First, update the following fields in the Scheduler Configuration of your YAML file:

Reviewer 4: ```suggestion
First, update the scheduler configuration (earlier you applied a
`my-scheduler.yaml` file that included a ConfigMap named `my-scheduler-config`; the
scheduler configuration is in YAML format, within the `data` field of that ConfigMap).

The fields to change in the inner KubeSchedulerConfiguration are:
```

To run multiple-scheduler with leader election enabled, you must do the following:

> First, update the scheduler configuration (earlier you applied a

Reviewer: See my [comment about scheduler configuration](https://github.com/kubernetes/website/pull/30107/files#r740745459).
In either case, we should keep the case the same on this page as 
>Scheduler Configuration

is upper camel case in other parts of the page

Author: @reylejano already updated with https://github.com/kubernetes/website/pull/30107/commits/c5cb03ab13e4e101875e03a61689cad458ec782f.

> First, update the scheduler configuration (earlier you applied a
> `my-scheduler.yaml` file that included a ConfigMap named `my-scheduler-config`; the
> scheduler configuration is in YAML format, within the `data` field of that ConfigMap).
> 
> The fields to change in the inner KubeSchedulerConfiguration are:

Reviewer: ```suggestion
Update the following fields for the KubeSchedulerConfiguration in the `my-scheduler-config` ConfigMap in your YAML file:
```

  {{< codenew file="admin/sched/pod3.yaml" >}}

  In this case, we specify that this pod should be scheduled using the scheduler that we
  deployed - `my-scheduler`. Note that the value of `spec.schedulerName` should match the name supplied for the scheduler

>   in the `schedulerName` field of the mapping `KubeSchedulerProfile` of the scheduler.

Reviewer 3: ```suggestion
  in the `schedulerName` field of the mapping `KubeSchedulerProfile`.
```
